pom.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <description>Simple SQL and JDBC educational project</description>
+
+    <groupId>com.cursor</groupId>
+    <artifactId>JDBCAndSQLOperation</artifactId>
```

Comment:
artifactId Should be lowercase. Words should be separated by "-".

Reply:
> artifactId Should be lowercase. Words should be separated by "-".

done